core/src/main/java/com/homeaway/streamplatform/streamregistry/health/EventStoreConfigHealthCheck.java

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2018-Present Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamplatform.streamregistry.health;
+
+import static org.apache.kafka.clients.consumer.ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG;
+
+import java.util.concurrent.ExecutionException;
+
+import com.homeaway.streamplatform.streamregistry.configuration.EventStoreTopic;
+import com.homeaway.streamplatform.streamregistry.utils.KafkaTopicClient;
+import lombok.extern.slf4j.Slf4j;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheck;
+
+import com.homeaway.streamplatform.streamregistry.configuration.StreamRegistryConfiguration;
+
+/**
+ * StreamRegistry uses Kafka as its Data-store.
```

Comment:
nit: This isn't necessarily always true. It only is when the infra provider uses Kafka.

Reply:
InfraProvider at this point only supports data for Kafka Clusters.

However,  the StreamMetaData is stored in Kafka